Library/Homebrew/formulary.rb

```diff
@@ -122,14 +122,14 @@ def initialize(bottle_name)
       super name, Formulary.path(full_name)
     end
 
-    def get_formula(spec, alias_path: nil)
-      formula = super
+    def get_formula(spec, alias_path: nil) # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
Is `# rubocop:disable Lint/UnusedMethodArgument` the best way to avoid this `brew style` error?
```
W:125: 27: Unused method argument - alias_path.
```
See https://github.com/bbatsov/rubocop/issues/3130

Reply:
Would you prefer…
```ruby
raise "Unexpected non-nil parameter alias_path" unless alias_path.nil?
```

Reply:
I think `def get_formula(spec, **)` should work.

Reply:
Yes, that works, but it does alter the behavior of the method for style, in that `get_formula(spec, monkey: :banana)` no longer fails. https://github.com/bbatsov/rubocop/issues/3130#issuecomment-274852535
My minor style preference is `raise … unless nil?`. I'm fine with `**` though.

Reply:
Done.